packages/components/src/Notification/ToastNotification/_docs/ToastNotification.mdx

```diff
@@ -17,51 +17,56 @@ import * as ToastNotificationStories from "./ToastNotification.stories"
 
 <Installation
   installCommand="yarn add @kaizen/components"
-  importStatement='import { addToastNotification, removeToastNotification, clearToastNotifications, ToastNotification } from "@kaizen/components"'
+  importStatement='import { addToastNotification, updateToastNotification, removeToastNotification, clearToastNotifications, ToastNotification } from "@kaizen/components"'
```

Comment:
This is a bit long, maybe we focus on the main function we want to champion? Was it `addToastNotification`? and move the rest of the imports to their respective sections.

Maybe even leave out the `<ToastNotification />` component to deter further use.